sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -121,15 +169,27 @@ namespace Azure { namespace Core { namespace Http {
      *
      */
     std::chrono::milliseconds ConnectionTimeout = _detail::DefaultConnectionTimeout;
+
+    /**
+     * @brief If set, enables extended tracing from libcurl.
```

Comment:
Is it extended tracing or tracing?  I'd call the option EnableCurlExtendedTracing if this is the case though I suspec this is more about just tracing in curl and thus dropping the 'extended' from the doc comment is the right change

Reply:
I've improved the comment, good catch.